ui/src/app/flags/analytics/Analytics.tsx

```diff
@@ -0,0 +1,140 @@
+import { Formik } from 'formik';
+import { useMemo, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useOutletContext } from 'react-router-dom';
+import { FetchBaseQueryError } from '@reduxjs/toolkit/query';
+import Combobox from '~/components/forms/Combobox';
+import 'chartjs-adapter-date-fns';
+import { addMinutes, format, parseISO } from 'date-fns';
+import { selectCurrentNamespace } from '~/app/namespaces/namespacesSlice';
+import { IFlag } from '~/types/Flag';
+import { BarGraph } from '~/components/graphs';
+import { IFilterable } from '~/types/Selectable';
+import Well from '~/components/Well';
+import { useGetFlagEvaluationCountQuery } from '~/app/flags/analyticsApi';
+
+type AnalyticsProps = {
+  flag: IFlag;
+};
+
+const timeFormat = 'yyyy-MM-dd HH:mm:ss';
+
+interface IDuration {
+  value: number;
+}
+
+type FilterableDurations = IDuration & IFilterable;
+
+const durations: FilterableDurations[] = [
+  {
+    value: 30,
+    key: '30 minutes',
+    displayValue: '30 minutes',
+    filterValue: '30 minutes'
+  },
+  {
+    value: 60,
+    key: '1 hour',
+    displayValue: '1 hour',
+    filterValue: '1 hour'
+  },
+  {
+    value: 60 * 4,
+    key: '4 hours',
+    displayValue: '4 hours',
+    filterValue: '4 hours'
+  },
+  {
+    value: 60 * 12,
+    key: '12 hours',
+    displayValue: '12 hours',
+    filterValue: '12 hours'
+  }
+];
+
+export default function Analytics() {
+  const [selectedDuration, setSelectedDuration] =
+    useState<FilterableDurations | null>(durations[0]);
+  const { flag } = useOutletContext<AnalyticsProps>();
+  const namespace = useSelector(selectCurrentNamespace);
+
+  const nowISO = parseISO(new Date().toISOString());
+
+  const getFlagEvaluationCount = useGetFlagEvaluationCountQuery({
+    namespaceKey: namespace.key,
+    flagKey: flag.key,
+    from: format(
+      addMinutes(
+        addMinutes(
+          nowISO,
+          selectedDuration?.value ? selectedDuration.value * -1 : -60
+        ),
+        nowISO.getTimezoneOffset()
+      ),
+      timeFormat
+    ),
+    to: format(addMinutes(nowISO, nowISO.getTimezoneOffset()), timeFormat)
+  });
+
+  const flagEvaluationCount = useMemo(() => {
+    const fetchError = getFlagEvaluationCount.error as FetchBaseQueryError;
+    return {
+      timestamps: getFlagEvaluationCount.data?.timestamps,
+      values: getFlagEvaluationCount.data?.values,
+      unavailable: fetchError?.status === 501
```

Comment:
another option is we could check the `/meta/config` endpoint to see if analytics are enabled before making the call and depending on a 501

![CleanShot 2024-02-07 at 10 18 46](https://github.com/flipt-io/flipt/assets/209477/1c4a265f-b5a6-4d41-baba-6c023db41b60)

I Could do this in a future PR though as it will require some redux work I think